code/src/index.css

```diff
@@ -1,13 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
```

Comment:
Very nice design, nice colour scheme and good balance with minimalist design without it feeling empty or boring. Nice font that goes perfectly with the rounded corners of the container and it works well in both mobile and desktop. The longer tasks dont have line break/word break but since the container is scrollable it still works so that clever!